syntax = "proto3";

service Predict {
    rpc predict (PredictRequest) returns (PredictResult) {}
}

message PredictRequest {
    string captcha_img = 1;
    string split_char = 2;
}

message PredictResult {
    string result = 1;
    int32 code = 2;
    bool success = 3;
}